posts/2019/11/Files-Improvements.md

```diff
@@ -0,0 +1,78 @@
+<!--
+.. title: Files Improvements Spyder4
+.. slug: Files-Improvements
+.. date: 2019-11-01 12:00:00 UTC-05:00
+.. author: Juanita Gomez & Gonzalo Peña
+.. tags: Labs, Spyder
+.. category: 
+.. link: 
+.. description: 
+.. type: text
+-->
+
+# Files Improvements Spyder4
+
+For version 4 of Spyder several improvements have been made regarding file management and visualization. 
```

Comment:
Start with "For Spyder 4.0"?

Reply:
Question about the "and visualization" - did you mean something specific? It doesn't really come back in the rest of the post. Is it the UI design of the file manager, or something else?

Reply:
I meant things like the file explorer simplified interface and the icons according to the file extensions (which I guess are part of the interface too). Maybe the word visualization could be replaced with "interface"? What do you think?

Reply:
The issue is at least in English in the data analysis field, the above sentence would almost invariably be read to refer to _data_ visualization (e.g. Plots panel, etc). "Interface" is closer to what you mean, but it cannot be grammatically used as is in this context without changing the sentence. I suggest just such a revision in the review I'm about to submit.